lib/template/templates.c

```diff
@@ -325,7 +324,7 @@ log_template_options_defaults(LogTemplateOptions *options)
 {
   memset(options, 0, sizeof(LogTemplateOptions));
   options->frac_digits = -1;
-  options->ts_format = TS_FMT_ISO;
+  options->ts_format = -1;
   options->on_error = -1;
```

Comment:
I don't get this patch or its original. I think ts_format needs to be set to -1 here and log_template_options_init() be called to initialize it.